bgbb_airflow/pred_airflow_job.py

```diff
@@ -52,6 +52,14 @@ def extract(
     param_prefix,
     model_win=90,
     sample_ids: Union[Tuple, List[int]] = (),
+    first_dims=[
```

Comment:
@acmiyaguchi now that I'm thinking about it, might it be a better idea to expose these at the cli level so that updates could be handled directly through telemetry-airflow instead of here?

Reply:
Either is fine, jobs in mozetl have generally kept configuration close to the code. Being able to parameterize stuff from airflow is pretty nice too though.